behind-the-docs-1.rst

```diff
@@ -0,0 +1,89 @@
+.. post:: July 1, 2021
+   :tags: behind-the-docs
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Learn more about Read the Docs directly from its team members:
+      appearances in the media during the first half of 2021.
+
+Behind the Docs episode 1
```

Comment:
Don't love this branding -- We should at least say `Behind the Scenes at Read the Docs` -- I want to get away from cutesy branding, and this definitely won't be obvious to users what it is :)

```suggestion
Behind the Scenes at Read the Docs
```

For something with more SEO value:

```suggestion
Early 2021 talks and podcasts from Read the Docs
```

Reply:
I knew you would _love_ it 😄 Accepting your second suggestion!